contrib/board_files/beaglebone.conf

```diff
@@ -1,3 +1,6 @@
+[board]
+model = TI AM335x BeagleBone Black
```

Comment:
Let me do a little bit of research to see if this is the same for all varients of beaglebone